app/assets/javascripts/application.js

```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', function () {
+  const migrationActions = document.querySelectorAll('.migration-action');
+
+  migrationActions.forEach(button => {
+    button.addEventListener('click', function (event) {
```

Comment:
Will the web server support multiple requests? Always? Consider the web server is configured to serve only one request at a time. How will it behave if the user clicks to run one migration and clicks another one without waiting for its response? Is it an acceptable solution to disable just the pressed button then?